tests/foreman/ui/test_host.py

```diff
@@ -2754,34 +2754,53 @@ def test_positive_set_multi_line_and_with_spaces_parameter_value(
 class TestHostAnsible:
     """Tests for Ansible portion of Hosts"""
 
-    @pytest.mark.stubbed
-    @pytest.mark.tier2
-    def test_positive_host_role_information(self):
-        """Assign Ansible Role to a Host and an attached Host group and verify that the information
-        in the new UI is displayed correctly
 
-        :id: 7da913ef-3b43-4bfa-9a45-d895431c8b56
+@pytest.mark.tier2
+def test_positive_host_role_information(target_sat):
+    """Assign Ansible Role to a Host and verify that the information
+    in the new UI is displayed correctly
```

Comment:
Why move this outside of the class? Seems like the test applies to that class.

Reply:
Not intentional, probably a merge/rebase thing that I messed up. I'll fix it.